scipy/linalg/_special_matrices.py

```diff
@@ -992,8 +1021,8 @@ def fiedler(a):
     """
     a = np.atleast_1d(a)
 
-    if a.ndim != 1:
-        raise ValueError("Input 'a' must be a 1D array.")
+    if a.ndim > 1:
+        return np.apply_along_axis(lambda a: fiedler(a), -1, a)
```

Comment:
Can this be simplified since the lambda doesnt change anything?
```suggestion
        return np.apply_along_axis(fiedler, -1, a)
```